<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QAircrack</class>
 <widget class="QWidget" name="QAircrack">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>288</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QAircrack</string>
  </property>
  <property name="windowIcon">
   <iconset resource="graphics.qrc">
    <normaloff>:/icons/icons/qaircrack.png</normaloff>:/icons/icons/qaircrack.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="0" colspan="2">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QPushButton" name="monitorButton">
        <property name="minimumSize">
         <size>
          <width>64</width>
          <height>64</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>64</width>
          <height>64</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="graphics.qrc">
          <normaloff>:/icons/icons/monitor-off.png</normaloff>:/icons/icons/monitor-off.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QPushButton" name="listButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>64</width>
          <height>64</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>64</width>
          <height>64</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="graphics.qrc">
          <normaloff>:/icons/icons/listar.png</normaloff>:/icons/icons/listar.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="7">
       <widget class="QPushButton" name="captureButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>64</width>
          <height>64</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>64</width>
          <height>64</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="graphics.qrc">
          <normaloff>:/icons/icons/capturar.png</normaloff>:/icons/icons/capturar.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="10">
       <widget class="QPushButton" name="authButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>64</width>
          <height>64</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>64</width>
          <height>64</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="graphics.qrc">
          <normaloff>:/icons/icons/autenticar.png</normaloff>:/icons/icons/autenticar.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="13">
       <widget class="QPushButton" name="inyectButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>64</width>
          <height>64</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>64</width>
          <height>64</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="graphics.qrc">
          <normaloff>:/icons/icons/inyectar.png</normaloff>:/icons/icons/inyectar.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="16">
       <widget class="QPushButton" name="keyButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>64</width>
          <height>64</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>64</width>
          <height>64</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="graphics.qrc">
          <normaloff>:/icons/icons/clave.png</normaloff>:/icons/icons/clave.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>48</width>
          <height>48</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="3">
       <widget class="QLabel" name="monitorLabel">
        <property name="text">
         <string>Activar
monitor</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <property name="margin">
         <number>6</number>
        </property>
       </widget>
      </item>
      <item row="1" column="3" colspan="3">
       <widget class="QLabel" name="listLabel">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Listar
redes</string>
        </property>
        <property name="scaledContents">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <property name="margin">
         <number>6</number>
        </property>
       </widget>
      </item>
      <item row="1" column="6" colspan="3">
       <widget class="QLabel" name="captureLabel">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Capturar</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="9" colspan="3">
       <widget class="QLabel" name="authLabel">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Autenticar</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="12" colspan="3">
       <widget class="QLabel" name="inyectLabel">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Inyectar</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="15" colspan="3">
       <widget class="QLabel" name="keyLabel">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Obtener
clave</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="3">
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="5">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="6">
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="8">
       <spacer name="horizontalSpacer_6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="9">
       <spacer name="horizontalSpacer_7">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="11">
       <spacer name="horizontalSpacer_8">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="12">
       <spacer name="horizontalSpacer_9">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="17">
       <spacer name="horizontalSpacer_10">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="15">
       <spacer name="horizontalSpacer_11">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="14">
       <spacer name="horizontalSpacer_12">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="myGroup">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Mi tarjeta wifi</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Interfaz</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="myInterface">
        <property name="text">
         <string>wlan0</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_12">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>MAC</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="myMac">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="myMonitor">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Monitor</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="myInterfaceButton">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="maximumSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="graphics.qrc">
          <normaloff>:/icons/icons/help.png</normaloff>:/icons/icons/help.png</iconset>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="apGroup">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Red wifi</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Nombre</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="apName">
        <property name="text">
         <string>cedepe</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>MAC</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="apMac">
        <property name="text">
         <string>00:23:CD:CE:5A:9A</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Canal</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="apChannel">
        <property name="text">
         <string>6</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="graphics.qrc"/>
 </resources>
 <connections/>
</ui>
